### PR TITLE
Ignore GSC verification in pa11y, change btn type to submit for availability form

### LINKED ISSUE #160: Hitting the enter key in the search box queries the domain availability API
### Current Behavior

See title. The domain availability field returns an error, too, since `.gov` is not available (it's true).

### Expected Behavior

Submitting text to the search box should return results from search.gov.

### Steps to Reproduce

1. Type anything in the top-line `search box` and hit `enter` (Clicking the search Icon still works properly)
2. After a moment, scroll down to see the domain availability error.

### Environment

beta.get.gov

### Additional Context
This is likely an effect of how we implemented #122. 

### Issue Links

_No response_

### LINKED ISSUE #213: Fix broken pa11y checks
### Issue description

When switching code over from the beta branch to main pa11y checks failed and we made the decision to merge anyways. This should be fixed promptly.

### Acceptance criteria

- [ ] all checks pass with no errors

### Additional context

_No response_

### Links to other issues

_No response_

### PR DESCRIPTION
**Pull request by rachidatecs**
## Ticket

Resolves #213 
Resolves #160 

## Changes

- Remove GSC verification file,
- Refactor availability form markup and js to allow form submission on Enter,
- Refactor scss for availability form,
- Refactor scss class names to use BEM convention,
- Update readme with instructions for clean/run and running pa11y.

#### Satisfied acceptance criteria and met development standards

- [ ] Met the acceptance criteria, or will meet them in a subsequent PR
- [ ] Created/modified automated tests
- [ ] Added at least 2 developers as PR reviewers (only 1 will need to approve)
- [ ] Messaged on Slack or in standup to notify the team that a PR is ready for review
- [ ] Changes to “how we do things” are documented in READMEs and or onboarding guide
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Original Developer)

- [ ] All new functions and methods are commented using plain language
- [ ] Did dependency updates in Pipfile also get changed in requirements.txt?
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values

#### Validated user-facing changes (if applicable)

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)
- [ ] Add at least 1 designer as PR reviewer

#### Reviewed, tested, and left feedback about the changes

- [ ] Pulled this branch locally and tested it
- [ ] Reviewed this code and left comments
- [ ] Checked that all code is adequately covered by tests
- [ ] Made it clear which comments need to be addressed before this work is merged
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Code reviewer)

- [ ] All new functions and methods are commented using plain language
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values
- [ ] (Rarely needed) Did dependency updates in Pipfile also get changed in requirements.txt?

#### Validated user-facing changes as a developer

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Meets all designs and user flows provided by design/product
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers, the suggestion is to use ones that the developer didn't (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

**Note:** Multiple code reviewers can share the checklists above, a second reviewers should not make a duplicate checklist

#### Verified that the changes match the design intention

- [ ] Checked that the design translated visually
- [ ] Checked behavior
- [ ] Checked different states (empty, one, some, error)
- [ ] Checked for landmarks, page heading structure, and links
- [ ] Tried to break the intended flow

#### Validated user-facing changes as a designer

- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

### CODE REVIEW

File: _includes/domain-search.html [hunk 1]
```diff
@@ -103,7 +96,7 @@ <h2>
                     <button 
                     id="check-domain-availability-button"
                     class="usa-button availability-button" 
-                    type="button"
+                    type="submit"
```
[the remark this answers is not in the document]
Comment (therealslimhsiehdy) [reply _includes/domain-search.html:99]: Needs to be moved `margin-left: 1 rem` .. looking into why the button edge is overflowing

[the remark this answers is not in the document]
Comment (therealslimhsiehdy) [reply _includes/domain-search.html:99]: Screenshot 2024-01-29 at 8 22 37 AM

These two `border` are culprit of the left edges being wonky

[the remark this answers is not in the document]
Comment (rachidatecs) [reply _includes/domain-search.html:99]: Cool, on it

**Review by neilmb — APPROVED** [review]
Lots of name changes, but that's probably good. LGTM

**Review by therealslimhsiehdy — APPROVED** [review]
LGTM!! You even got the search button fix in too YOU THE WIZARD RACHID 🥳 Thank you for your patience with all the scss updates!

**Comment by rachidatecs** [comment]
New PR with signed commits [here](https://github.com/cisagov/get.gov/pull/216)

**Pull request closed without merging**